index.php

```diff
@@ -0,0 +1,35 @@
+<html>
+<head>
+    <title>Welcome to website</title>
+    <link rel='stylesheet' href="style.css?Time=<?php echo microtime()?>" />
+</head>
+
+<body>
+<div class="headerbackgroundimage">
+        <ul>
+            <li><a class="active" href="index.php">home</a></li>
+            <li><a href="contact.php">say hello</a></li>
+            <li><a href="resume.php">resume</a></li>
+            <li><a href="Myblog.php">blog + inspiration</a></li>
+            <li><a href="Moreinfo.php">about</a></li>
+        </ul>
+    <h1> Hi, I'm Jillian. </h1>
+</div>
+<div class="welcomeboxes">
+    <div class="box1">
+    <h2>welcome!</h2>
+    <p>
+        My name is Jillian. I'm a student at Washington University learning how to code this summer in stl. I'm glad you're here :-)
+</div>
+<div class="box2">
+    <h3>what I'm listening to</h3>
+    <p>
+        spotify</p>
+    </div>
+        <div class="box3">
```

Comment:
Calling something `box3` works technically, but I'd suggest using a more descriptive name.

Although in this case, it looks like the design of these boxes is the same (or very close anyway). Could you give all three divs the same class so you don't need three of them?